### PR TITLE
Add options on interstitial 1

### PR DESCRIPTION
**Pull request by emmabeynon**
For [Trello](https://trello.com/c/KTbbRrYD/69-2-build-the-prototype-compare-different-ways-to-access-a-service-on-the-interstitial-page)

This is the first prototype for displaying sign-in options on the interstitial page for Check your state pension.

Outstanding actions
- [ ] Add missing copy (outlined in Trello card)

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -7,5 +7,14 @@ router.get('/', function (req, res) {
 })
 
 // add your routes here
+router.post('/check-state-pension-options-1', function (req, res) {
+  if (req.body.loginoption === "government-gateway") {
+    res.redirect('https://www.tax.service.gov.uk/gg/sign-in?continue=%2Fcheck-your-state-pension%2Faccount&origin=nisp-frontend&accountType=individual')
```
Comment (nickcolley) [inline app/routes.js]: This isn't a pattern you see often, a form redirecting you to another domain, I wonder if there's any possibility for this to confuse users.

File: app/views/options-on-interstitial/check-state-pension-1.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Start page example
```
Comment (NickColley) [inline app/views/options-on-interstitial/check-state-pension-1.html]: Users of assistive tech use titles to know what page they have landed on, we probably need to think what this should be.

File: app/views/options-on-interstitial/check-state-pension-1.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        <p>
+          You can’t use this service if you’re already getting your State Pension or if you’ve delayed (‘deferred’) claiming it.
+        </p>
+      </div>
+
+      <h2 class="heading-medium">You need to sign in</h2>
+      <p>
+        Lorem ipsum dolor sit amet, te vix nisl splendide signiferumque, mel reque appetere cu. Pro eu populo conceptam. Id copiosae dissentias eam. Eum ea dico exerci dicunt, mea te habeo.
+      </p>
+      <a class="button" type="submit" href="./check-state-pension-interstitial-1">Sign in</a>
```
Comment (NickColley) [inline app/views/options-on-interstitial/check-state-pension-1.html]: type="submit" is used on the `<input>` tag, here we could use `role="button"` instead to indicate that these are buttons.

File: app/views/options-on-interstitial/check-state-pension-interstitial-1.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      <p>
+        Lorem ipsum dolor sit amet, te vix nisl splendide signiferumque, mel reque appetere cu. Pro eu populo conceptam. Id copiosae dissentias eam. Eum ea dico exerci dicunt, mea te habeo.
+      </p>
+    </div>
+  </div>
+  <form role="form" action="/check-state-pension-options-1" method="post" id="login-options">
+    <div class="form-group">
+      <fieldset>
+        <div class="multiple-choice">
+          <input id="government-gateway" type="radio" name="loginoption" value="government-gateway"/>
```
Comment (NickColley) [inline app/views/options-on-interstitial/check-state-pension-interstitial-1.html]: At the moment when you interact with these radios, it's not clear what they're for.

To give context we need to move the heading so it's within the `fieldset` and within a `legend` this means when a user of assistive tech interacts with this form, the heading will be read out giving context to what they're doing.

**Review by NickColley — COMMENTED** [review]
.

File: app/views/options-on-interstitial/check-state-pension-1.html (hunk 2 above)
Comment (emmabeynon) [reply app/views/options-on-interstitial/check-state-pension-1.html]: I've changed this to "Check your state pension".

File: app/views/options-on-interstitial/check-state-pension-1.html (hunk 3 above)
Comment (emmabeynon) [reply app/views/options-on-interstitial/check-state-pension-1.html]: I've changed this to `role="button"` 👍

File: app/views/options-on-interstitial/check-state-pension-interstitial-1.html (hunk 4 above)
Comment (emmabeynon) [reply app/views/options-on-interstitial/check-state-pension-interstitial-1.html]: I've had a go at addressing this - can you take a look to see if it's what you were expecting?

**Pull request merged**